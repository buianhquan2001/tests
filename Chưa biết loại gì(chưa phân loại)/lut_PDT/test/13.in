14
39 23
             #    #              #    #
    #        ##   #   #          # #  #
    #        ##   #   #          # #  #
    #        ##   ##  #          # #  #
   ##     #  ##   ##  #          # #  #
   ##     #  ##   ##  #   #    # # #  #
   ##     #  ##   ### #   #    # # #  #
   ##     #  ##   ### #   #    # # ## #
   ##     #  ##   ### #  ##    ### ## #
   ##     #  ###  ### #  ##    ### ## #
#  ##     ## ###  ### # ###    ### ## #
#  ####   ## ###  ### # ###    ### ## #
#  ####   ## ###  ### # ### ## ### ## #
#  ####   ## ###  ### # ### ###### ## #
#  #### # ## ###  ### # ### ###### ## #
#  ###### ## #### ##### ### ###### ## #
#  ###### ## #### ##### ### ###### ## #
#  ######### ########## ############# #
#  ######### ########## ############# #
#  ######### ######################## #
# ################################### #
##################################### #
#######################################
58 21
     ##   #   # #                                     #   
     ##   #   # #                        #    #       #   
     ## # #   # #                 # #    #    #       #  #
     ## # #   # #                 # #    #    #   #   #  #
     #### #  ## #       #       # # #    # #  #   #   #  #
     #### #  ## #       #       # # ##   # #  #   #   #  #
 #   #### #  ## #       ##   #  # # ##   # #  #   #   #  #
 ##  #### # ### #       ##   #  # # ##   # #  #   # # #  #
 ##  #### # ### #     # ###  #  # ####   # #  #   # # #  #
 ##  #### # ### # #   # ###  #  # #####  # ## #   # # ## #
 ##  #### # ### # #  ## #### #  # #####  # ## #   # # ## #
 ##  #### # ### # #  ## #### #  # #####  # ## #   # # ## #
 ##  #### # ### # #  ## #### #  # ###### #### ##  # # ## #
 ##  #### # ### # ## ## #### # ## ###### #### ##  # # ## #
 ##  #### # ### # ## ## #### # ## ###### #### ##  # # ## #
 ##  ###### ### # ## ## #### # ## ###### #### ## ## # ## #
 ##  ###### ### # ########## ########### #### ## ## # ## #
 ##  ###### ### # ########## ########### #### ##### # ## #
 ### ###### ### # ########################### ##### #### #
#### ###### ##############################################
##########################################################
44 24
         #                                  
         #         #      #                 
         #         #      #                #
      #  #         #      #                #
 #    #  #     #   # #    #        #   #   #
 #    #  ##    #   # #    #    #   #   #   #
 #    #  ##    #   ###    #    #   #   #   #
 #    #  ##    #   ###    #    #   ##  #   #
 #    #  ##    ##  ###    #    #   ##  #   #
 #    #  ##    ##  ###    #    #   ##  ##  #
 #   ##  ##    ##  ###    #    # # ## ###  #
 #   ##  ##    ##  ###    #    # # ## ###  #
 #   ##  ##    ##  ###    #    # # ## #### #
 #   ##  ##    ### ###    #    # # ####### #
 #   ##  ##    ### ####   #    # # ####### #
 ##  ##  ##    ### ####   #    # # ####### #
 ## ###  ### # ### ####   #    # # #########
 ## #### ##### ### ####   ##   # # #########
######## ##### ### ###### #### # # #########
######## ######### ########### # ###########
######## ######### ########### #############
############################## #############
############################################
############################################
67 26
                 #        #     # #      #       #                 
   #             #        #     # #      #       #    #            
   #             # # #    # #   # #      #       #    #            
   #         #   # # ##   # #  ## #  #   #       #    #            
#  #         #   # # ##   # #  ## #  #   #       #    #   #        
#  #         #   # # ###  # #  ## #  #   #       #    #   #        
#  #         #   # # ###  ###  ## #  #   #       #    # # #        
#  #         #   # # #### ###  ## #  #   ##      #    # # #        
#  # ##      #   # # #### ###  ## # ##   ##      #    # # #        
#  # ##     ##   # # #### #### ## # ##   ##     ##    # # ##       
#  # ##     ##   # # #### ####### # ##   ##     ##    # # ##       
#  # ## ##  ## # ### #### ####### # ##  ###     ##    # # ##   # # 
#  # ## ##  ## # ### #### ####### # ##  ###     ##    # # ##   # ##
# ## ## ##  ## # ### #### ####### # ##  ####    ##    # # ##   # ##
# ## ## ##  ## # ### #### ####### # ##  ####    ##    # # ##   # ##
# ##### ##  ## ##### #### ######### ##  ####    ###   # # ##   # ##
# ##### ## ### ##### #### ######### ##  ####    ###   # # ##   # ##
####### ## ### ##### #### ######### ##  ####    ###   # # ##   # ##
####### ## ### ##### ############## ##  ####    ###  ## # ##   # ##
####### ## ### ##### ############## ##  ####    ###  ## #### # # ##
####### ## ### ##### ############## ##  ####   ####  ## #### # # ##
####### ## ### ##### ############## ########   ####  ## #### # ####
####### ###### ##### #######################   ####  ## #### ######
####### ############ ######################## ###### ## ###########
#################### ############################### ## ###########
###################################################################
56 26
   #   #                                          #    #
   #   #                       #                  #    #
   #   #                       # #   #            #    #
   #   #                       # #   #            #    #
   #   #                       # #   #            #    #
   #   #    #      #    #  #   # #  ###      ##   #    #
   #   #    #      #    #  #   # #  ###      ##  ##    #
   #   #    # #   ##    #  #   # #  ###      ##  ##    #
   #   #  # # #   ##  # #  #   # #  ###      ##  ##    #
 # #   #  # # #   ##  # #  #   # ## ###      ##  ##    #
 # #   #  ### #   ##  # #  #   # ## ###      ##  ##    #
 # #   #  ### # # ##  # #  #   # ## ###    # ##  ##    #
 # #   # #### # # ##  # #  #   # ## ###    # ##  ##   ##
 # #   ###### # # ##### #  #   # ## ###    # ##  ##   ##
 # #   ###### # # ##### #  #   #### ###    # ##  ##  ###
 # #   ###### # # ##### #  #   #### ###    # ##  ##  ###
 # ### ###### # # ##### #  #  ##### ###    # ##  ##  ###
 # ### ###### # # ##### #  #  ##### ### #  # ##  ### ###
 # ### ###### # # ##### #  #  ##### ### #  # ##  ### ###
## ### ###### # # ##### #  #  ##### ### #  # ##  ### ###
## ### ###### ### ##### #  # ###### ### #  # ##  ### ###
## ### ###### ### ##### #  ######## #####  # ##  ### ###
## ### ###### ######### #  ######## #####  # ##  ### ###
## ### ###### ######### #  ############## ## ##  ### ###
########################## #################### #### ###
########################################################
46 20
                #     #     #   #          #  
                #     #     #   #     #    #  
               ###  # #     #   # #   ##   #  
            #  ###  # #     #   # #   ##   #  
           ## ####  # #     #   # #   ##   #  
        #  ## ####  # #     #   # #   ##   #  
        #  ## ##### # #     ##  # #   ##   ## 
        #  ## ##### # #  #  ##  # #   ##   ## 
 #      #  ## ####### #  #  ##  ### # ## # ###
 ##     #  ## ####### #  ## ##  ### # ## # ###
 ####   ## ## ####### #  ## ##  ### # ## # ###
 ####   ## ## ####### #  ###### ### # ## # ###
 ####   ## ## ####### #  ###### ##### ## # ###
#####   ## ## #########  ###### ##### ## # ###
######  ## ## ########## ###### ##### #### ###
######  ## ############# ################# ###
######  ## ############# ################# ###
###### ### ############# #####################
###### ################# #####################
##############################################
31 23
                      #        
                   #  #        
     #    #        #  #        
    ##    #        #  #        
    ##    #      # #  #        
    ## #  #      # ####  #    #
    ## #  #  #   # ####  #    #
    ## #  #  #   # ####  #    #
    ## #  #  #   # ####  #    #
##  ####  #  #   # ####  #    #
##  ####  #  #   # ####  #  # #
##  ####  #  #   ###### ##  # #
##  ####  ## # # ###### ##  # #
##  ####  ## # # ###### ##  # #
##  ####  ## ### ###### ### # #
##  ####  ## ### ###### ### # #
## #####  ## ### ###### ### ###
## ###### ## ### ########## ###
## ###### ## ### ########## ###
## ###### ## ### ########## ###
## ######### ### ########## ###
############ ### ########## ###
###############################
39 23
                             #     #   
      ##  #                  #     #   
      ##  #                  #     #   
      ### #   #              #     #   
#     ### #   #             ##     #   
#     ### ##  #       #  #  ##     #   
##    ### ##  #       #  #  ##     # # 
##    ### ##  #       #  ## ##     # # 
##   #### ##  #       # ### ##     # # 
##   #### ##  #       # ### ##     # ##
##   #### ##  #       ##### ##     # ##
###  ######## ###     ##### ##     ####
###  ######## ###     ##### ##     ####
#### ######## ####   ###### ##     ####
#### ######## #####  ###### ##     ####
#### ######## #####  #########     ####
#### ######## #####  ########## # #####
#### ######## ###### ########## # #####
#### ######## ###### ############ #####
#### ######## ################### #####
#### ##################################
#### ##################################
#######################################
55 30
                                               #       
                             #                 #       
    #         #              #                ##       
    #         # #            #                ##       
    ##        # #            #                ##       
    ##        # #            #       #        ##       
   ###     #  # #     #      #   #   #        ## #     
 # ###  #  #  # #     #      #   #   #        ## #     
 # ###  #  #  # #     #  ##  #   #   #    #   ####   # 
 # ###  #  #  # #     # ###  #   #   #    #   ####   # 
 # ###  #  #  # #    ## ###  #   #   #    #   ####   # 
 # ###  #  # ## #    ## ###  #   #   #  # #   ####  ## 
 # ###  #  # ## #   ### ###  #   #   #  # ##  ####  ## 
 # ###  #  # ## #   ### ###  #   #   #  # ##  ####  ## 
 # ###  #  # ## #   ### ###  #   #   #  # ##  ####  ## 
 # ###  #  # ## ##  ### ###  #   #   #  # ##  ####  ## 
 # ###  #  # ## ##  ### ###  #   #   #  # ##  ####  ## 
 # ###  #  # ## ##  ### ###  #  ##   #  ####  ####  ## 
 # ###  #  # ## ##  ### ###  #  ##   #  ####  ##### ## 
 # ### ##  # #####  ### ###  #  ###  #  ####  ##### ## 
 ##### ##  # ##### #### ### ##  ###  #  ####  ##### ## 
 ##### ##  # ########## ### ### ###  #  ####  ##### ## 
 ##### ### # ########## ### ### ###  #  ##### ##### ###
 ##### ### # ########## ### #######  #  ##### ##### ###
###### ### # ########## ### #######  #  ##### ##### ###
###### ### # ########## ### ######## #  ###############
####################### ############ #  ###############
####################### ############## ################
####################### ############## ################
#######################################################
36 25
    ##                           #  
    ## # #                       ## 
#   ## # #                       ## 
# # ## # #  #                    ## 
# # ## # #  #    #   #           ## 
# # ## # #  #    #   ##          ## 
# # ## # #  #    #   ## #        ## 
### ## # ## #    #   ## #     #  ###
### ## # ## ##   #   ## #   ### ####
### ## # ## ##   #  ### #   ### ####
### ## # ## ##   #  ### ##  ### ####
### ## # ## ##   #  ######  ### ####
### ## # ## ##   #  ######  ### ####
### ## # ## ##   #  ######  ### ####
### ## # ## ##   #  ###### #### ####
######## ## ##   #  ###### #### ####
######## ## ###  #  ###### #### ####
######## ## ###  #  ###### #### ####
######## ## ###  ## ###### #########
######## ######  ## ###### #########
######## ######  ## ################
######## ######  ## ################
######## ###### ####################
####################################
####################################
31 20
#                 #       #    
#                 #       #    
# #               #       #   #
# #           #   #       #   #
# #           #   # # #  ##   #
# #       #   #   # # #  ##   #
# # ##    #   ##  # # ## ##   #
# # ##    #   ## ## # ## ##   #
# # ##    #   ## ## #### ##  ##
# # ##    #   ## ## #### ##  ##
# # ##    # # ## ## #### ## ###
# # ##    # # ## ## #### ######
# # ##  # ### ##### #### ######
# # ##  # ### ##### ###########
# # ##  # ### ##### ###########
# # ### # #####################
# # ### # #####################
# # ###########################
# # ###########################
###############################
42 26
   ##                     # #       #     
   ## #     #             # #       #     
   ## #     #             # #       #     
   ## #     #             # #       #     
   ## #     #             # #       #     
 # ## #   ###             # #  #    #     
 # ## #   ###             # #  #  # ##    
 # ## #   ###             # #  #  # ##    
 # ## #   ###             # #  ## # ##  # 
 # ## #   ### #   #       # ## ## # ##  ##
 # ## #   ### #   #       # ## ## # ##  ##
 # ## #   ### #   #       # ## ## # ## ###
 # ## #   ### #   #       #### ## # ## ###
 # ## # # ### # # ##      #### ## # ## ###
 #### # # ### # ####      #### ## # ## ###
 #### # # ### # ####      #### ## # ## ###
##### # # ### # ####    # #### ## # ## ###
##### # # ### # ####### # #### ## # ## ###
##### # # ### # ####### # #### ## # ## ###
##### # # ##### ############## ## # ## ###
####### # ##### ############## ## ########
####### # ##### ############## ## ########
####### # ##### ############## ## ########
####### # #################### ## ########
######### ####################### ########
##########################################
34 23
 #               #                
 #       #  #    #                
 #       #  #    #                
 #      ##  #    #        #      #
 #      ## ##    #        #     ##
 #      ## ###  ##        #     ##
 #      ## ###  ##        # #   ##
##      ## ###  ##        # #   ##
##      ## ###  ##        # #   ##
##      ## ###  ##        # #   ##
##      ## ###  ##       ## #  ###
## #   ### ###  ###   ## ## #  ###
## #   ### ###  ###   ## ## #  ###
## #   ### ###  ###   ## ## ## ###
## #   ### ###  ###   ## ## ## ###
## #   ### ###  ###   ## ## ## ###
## #   ### ###  #### ### ## ## ###
## #   #######  #### ### ## ## ###
## #  ########  #### ### ## ## ###
## #  ########  ######## #########
## ########### ######### #########
## ###############################
##################################
52 20
     #     #     #        #    ##                   
#    #     #     #        #    ##       #           
#    #     #     #    ##  #    ## #     #  #        
#    #  ## #     #    ##  #    ## #     # ##    #   
#    ## ## #     # #  ##  #    ## #    #####    #  #
#    ## ## #     # #  ##  #    ## #  # #####    #  #
#    ## ## #     ###  ##  #    ## # ## #####    #  #
#    ## ## #   # ###  ##  #    ## # ## #####   ##  #
#    ##### #   # ###  ## ##    ## # ## ######  ##  #
#    ##### ##  # #### ## ##    ## # ## ###### ###  #
#    ##### ##  ###### ## ##    ## # ## ########## ##
##   ##### ##  ###### ## ##  # ## # ## ########## ##
##   ##### ## ####### ## ##  # ## # ## #############
##   ##### ## ####### #####  # ## # ################
###  ##### ## ####### #####  #### ##################
###  ##### ## ####### #####  #### ##################
#### ######## ####### ###### #### ##################
#### ################ ###### #### ##################
#### ###############################################
####################################################
