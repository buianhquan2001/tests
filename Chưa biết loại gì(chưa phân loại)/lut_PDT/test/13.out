             #----#--------------#----#
    #--------##---#---#----------#-#--#
    #--------##---#---#----------#-#--#
    #--------##---##--#----------#-#--#
   ##-----#--##---##--#----------#-#--#
   ##-----#--##---##--#---#----#-#-#--#
   ##-----#--##---###-#---#----#-#-#--#
   ##-----#--##---###-#---#----#-#-##-#
   ##-----#--##---###-#--##----###-##-#
   ##-----#--###--###-#--##----###-##-#
#--##-----##-###--###-#-###----###-##-#
#--####---##-###--###-#-###----###-##-#
#--####---##-###--###-#-###-##-###-##-#
#--####---##-###--###-#-###-######-##-#
#--####-#-##-###--###-#-###-######-##-#
#--######-##-####-#####-###-######-##-#
#--######-##-####-#####-###-######-##-#
#--#########-##########-#############-#
#--#########-##########-#############-#
#--#########-########################-#
#-###################################-#
#####################################-#
#######################################
338
     ##---#---#-#-------------------------------------#   
     ##---#---#-#------------------------#----#-------#   
     ##-#-#---#-#-----------------#-#----#----#-------#--#
     ##-#-#---#-#-----------------#-#----#----#---#---#--#
     ####-#--##-#-------#-------#-#-#----#-#--#---#---#--#
     ####-#--##-#-------#-------#-#-##---#-#--#---#---#--#
 #---####-#--##-#-------##---#--#-#-##---#-#--#---#---#--#
 ##--####-#-###-#-------##---#--#-#-##---#-#--#---#-#-#--#
 ##--####-#-###-#-----#-###--#--#-####---#-#--#---#-#-#--#
 ##--####-#-###-#-#---#-###--#--#-#####--#-##-#---#-#-##-#
 ##--####-#-###-#-#--##-####-#--#-#####--#-##-#---#-#-##-#
 ##--####-#-###-#-#--##-####-#--#-#####--#-##-#---#-#-##-#
 ##--####-#-###-#-#--##-####-#--#-######-####-##--#-#-##-#
 ##--####-#-###-#-##-##-####-#-##-######-####-##--#-#-##-#
 ##--####-#-###-#-##-##-####-#-##-######-####-##--#-#-##-#
 ##--######-###-#-##-##-####-#-##-######-####-##-##-#-##-#
 ##--######-###-#-##########-###########-####-##-##-#-##-#
 ##--######-###-#-##########-###########-####-#####-#-##-#
 ###-######-###-#-###########################-#####-####-#
####-######-##############################################
##########################################################
507
         #                                  
         #---------#------#                 
         #---------#------#----------------#
      #--#---------#------#----------------#
 #----#--#-----#---#-#----#--------#---#---#
 #----#--##----#---#-#----#----#---#---#---#
 #----#--##----#---###----#----#---#---#---#
 #----#--##----#---###----#----#---##--#---#
 #----#--##----##--###----#----#---##--#---#
 #----#--##----##--###----#----#---##--##--#
 #---##--##----##--###----#----#-#-##-###--#
 #---##--##----##--###----#----#-#-##-###--#
 #---##--##----##--###----#----#-#-##-####-#
 #---##--##----###-###----#----#-#-#######-#
 #---##--##----###-####---#----#-#-#######-#
 ##--##--##----###-####---#----#-#-#######-#
 ##-###--###-#-###-####---#----#-#-#########
 ##-####-#####-###-####---##---#-#-#########
########-#####-###-######-####-#-#-#########
########-#########-###########-#-###########
########-#########-###########-#############
##############################-#############
############################################
############################################
430
                 #--------#-----#-#------#-------#                 
   #-------------#--------#-----#-#------#-------#----#            
   #-------------#-#-#----#-#---#-#------#-------#----#            
   #---------#---#-#-##---#-#--##-#--#---#-------#----#            
#--#---------#---#-#-##---#-#--##-#--#---#-------#----#---#        
#--#---------#---#-#-###--#-#--##-#--#---#-------#----#---#        
#--#---------#---#-#-###--###--##-#--#---#-------#----#-#-#        
#--#---------#---#-#-####-###--##-#--#---##------#----#-#-#        
#--#-##------#---#-#-####-###--##-#-##---##------#----#-#-#        
#--#-##-----##---#-#-####-####-##-#-##---##-----##----#-#-##       
#--#-##-----##---#-#-####-#######-#-##---##-----##----#-#-##       
#--#-##-##--##-#-###-####-#######-#-##--###-----##----#-#-##---#-# 
#--#-##-##--##-#-###-####-#######-#-##--###-----##----#-#-##---#-##
#-##-##-##--##-#-###-####-#######-#-##--####----##----#-#-##---#-##
#-##-##-##--##-#-###-####-#######-#-##--####----##----#-#-##---#-##
#-#####-##--##-#####-####-#########-##--####----###---#-#-##---#-##
#-#####-##-###-#####-####-#########-##--####----###---#-#-##---#-##
#######-##-###-#####-####-#########-##--####----###---#-#-##---#-##
#######-##-###-#####-##############-##--####----###--##-#-##---#-##
#######-##-###-#####-##############-##--####----###--##-####-#-#-##
#######-##-###-#####-##############-##--####---####--##-####-#-#-##
#######-##-###-#####-##############-########---####--##-####-#-####
#######-######-#####-#######################---####--##-####-######
#######-############-########################-######-##-###########
####################-###############################-##-###########
###################################################################
664
   #---#------------------------------------------#----#
   #---#-----------------------#------------------#----#
   #---#-----------------------#-#---#------------#----#
   #---#-----------------------#-#---#------------#----#
   #---#-----------------------#-#---#------------#----#
   #---#----#------#----#--#---#-#--###------##---#----#
   #---#----#------#----#--#---#-#--###------##--##----#
   #---#----#-#---##----#--#---#-#--###------##--##----#
   #---#--#-#-#---##--#-#--#---#-#--###------##--##----#
 #-#---#--#-#-#---##--#-#--#---#-##-###------##--##----#
 #-#---#--###-#---##--#-#--#---#-##-###------##--##----#
 #-#---#--###-#-#-##--#-#--#---#-##-###----#-##--##----#
 #-#---#-####-#-#-##--#-#--#---#-##-###----#-##--##---##
 #-#---######-#-#-#####-#--#---#-##-###----#-##--##---##
 #-#---######-#-#-#####-#--#---####-###----#-##--##--###
 #-#---######-#-#-#####-#--#---####-###----#-##--##--###
 #-###-######-#-#-#####-#--#--#####-###----#-##--##--###
 #-###-######-#-#-#####-#--#--#####-###-#--#-##--###-###
 #-###-######-#-#-#####-#--#--#####-###-#--#-##--###-###
##-###-######-#-#-#####-#--#--#####-###-#--#-##--###-###
##-###-######-###-#####-#--#-######-###-#--#-##--###-###
##-###-######-###-#####-#--########-#####--#-##--###-###
##-###-######-#########-#--########-#####--#-##--###-###
##-###-######-#########-#--##############-##-##--###-###
##########################-####################-####-###
########################################################
703
                #-----#-----#---#----------#  
                #-----#-----#---#-----#----#  
               ###--#-#-----#---#-#---##---#  
            #--###--#-#-----#---#-#---##---#  
           ##-####--#-#-----#---#-#---##---#  
        #--##-####--#-#-----#---#-#---##---#  
        #--##-#####-#-#-----##--#-#---##---## 
        #--##-#####-#-#--#--##--#-#---##---## 
 #------#--##-#######-#--#--##--###-#-##-#-###
 ##-----#--##-#######-#--##-##--###-#-##-#-###
 ####---##-##-#######-#--##-##--###-#-##-#-###
 ####---##-##-#######-#--######-###-#-##-#-###
 ####---##-##-#######-#--######-#####-##-#-###
#####---##-##-#########--######-#####-##-#-###
######--##-##-##########-######-#####-####-###
######--##-#############-#################-###
######--##-#############-#################-###
######-###-#############-#####################
######-#################-#####################
##############################################
272
                      #        
                   #--#        
     #----#--------#--#        
    ##----#--------#--#        
    ##----#------#-#--#        
    ##-#--#------#-####--#----#
    ##-#--#--#---#-####--#----#
    ##-#--#--#---#-####--#----#
    ##-#--#--#---#-####--#----#
##--####--#--#---#-####--#----#
##--####--#--#---#-####--#--#-#
##--####--#--#---######-##--#-#
##--####--##-#-#-######-##--#-#
##--####--##-#-#-######-##--#-#
##--####--##-###-######-###-#-#
##--####--##-###-######-###-#-#
##-#####--##-###-######-###-###
##-######-##-###-##########-###
##-######-##-###-##########-###
##-######-##-###-##########-###
##-#########-###-##########-###
############-###-##########-###
###############################
217
                             #-----#   
      ##--#------------------#-----#   
      ##--#------------------#-----#   
      ###-#---#--------------#-----#   
#-----###-#---#-------------##-----#   
#-----###-##--#-------#--#--##-----#   
##----###-##--#-------#--#--##-----#-# 
##----###-##--#-------#--##-##-----#-# 
##---####-##--#-------#-###-##-----#-# 
##---####-##--#-------#-###-##-----#-##
##---####-##--#-------#####-##-----#-##
###--########-###-----#####-##-----####
###--########-###-----#####-##-----####
####-########-####---######-##-----####
####-########-#####--######-##-----####
####-########-#####--#########-----####
####-########-#####--##########-#-#####
####-########-######-##########-#-#####
####-########-######-############-#####
####-########-###################-#####
####-##################################
####-##################################
#######################################
316
                                               #       
                             #-----------------#       
    #---------#--------------#----------------##       
    #---------#-#------------#----------------##       
    ##--------#-#------------#----------------##       
    ##--------#-#------------#-------#--------##       
   ###-----#--#-#-----#------#---#---#--------##-#     
 #-###--#--#--#-#-----#------#---#---#--------##-#     
 #-###--#--#--#-#-----#--##--#---#---#----#---####---# 
 #-###--#--#--#-#-----#-###--#---#---#----#---####---# 
 #-###--#--#--#-#----##-###--#---#---#----#---####---# 
 #-###--#--#-##-#----##-###--#---#---#--#-#---####--## 
 #-###--#--#-##-#---###-###--#---#---#--#-##--####--## 
 #-###--#--#-##-#---###-###--#---#---#--#-##--####--## 
 #-###--#--#-##-#---###-###--#---#---#--#-##--####--## 
 #-###--#--#-##-##--###-###--#---#---#--#-##--####--## 
 #-###--#--#-##-##--###-###--#---#---#--#-##--####--## 
 #-###--#--#-##-##--###-###--#--##---#--####--####--## 
 #-###--#--#-##-##--###-###--#--##---#--####--#####-## 
 #-###-##--#-#####--###-###--#--###--#--####--#####-## 
 #####-##--#-#####-####-###-##--###--#--####--#####-## 
 #####-##--#-##########-###-###-###--#--####--#####-## 
 #####-###-#-##########-###-###-###--#--#####-#####-###
 #####-###-#-##########-###-#######--#--#####-#####-###
######-###-#-##########-###-#######--#--#####-#####-###
######-###-#-##########-###-########-#--###############
#######################-############-#--###############
#######################-##############-################
#######################-##############-################
#######################################################
631
    ##---------------------------#  
    ##-#-#-----------------------## 
#---##-#-#-----------------------## 
#-#-##-#-#--#--------------------## 
#-#-##-#-#--#----#---#-----------## 
#-#-##-#-#--#----#---##----------## 
#-#-##-#-#--#----#---##-#--------## 
###-##-#-##-#----#---##-#-----#--###
###-##-#-##-##---#---##-#---###-####
###-##-#-##-##---#--###-#---###-####
###-##-#-##-##---#--###-##--###-####
###-##-#-##-##---#--######--###-####
###-##-#-##-##---#--######--###-####
###-##-#-##-##---#--######--###-####
###-##-#-##-##---#--######-####-####
########-##-##---#--######-####-####
########-##-###--#--######-####-####
########-##-###--#--######-####-####
########-##-###--##-######-#########
########-######--##-######-#########
########-######--##-################
########-######--##-################
########-######-####################
####################################
####################################
329
#-----------------#-------#    
#-----------------#-------#    
#-#---------------#-------#---#
#-#-----------#---#-------#---#
#-#-----------#---#-#-#--##---#
#-#-------#---#---#-#-#--##---#
#-#-##----#---##--#-#-##-##---#
#-#-##----#---##-##-#-##-##---#
#-#-##----#---##-##-####-##--##
#-#-##----#---##-##-####-##--##
#-#-##----#-#-##-##-####-##-###
#-#-##----#-#-##-##-####-######
#-#-##--#-###-#####-####-######
#-#-##--#-###-#####-###########
#-#-##--#-###-#####-###########
#-#-###-#-#####################
#-#-###-#-#####################
#-#-###########################
#-#-###########################
###############################
260
   ##---------------------#-#-------#     
   ##-#-----#-------------#-#-------#     
   ##-#-----#-------------#-#-------#     
   ##-#-----#-------------#-#-------#     
   ##-#-----#-------------#-#-------#     
 #-##-#---###-------------#-#--#----#     
 #-##-#---###-------------#-#--#--#-##    
 #-##-#---###-------------#-#--#--#-##    
 #-##-#---###-------------#-#--##-#-##--# 
 #-##-#---###-#---#-------#-##-##-#-##--##
 #-##-#---###-#---#-------#-##-##-#-##--##
 #-##-#---###-#---#-------#-##-##-#-##-###
 #-##-#---###-#---#-------####-##-#-##-###
 #-##-#-#-###-#-#-##------####-##-#-##-###
 ####-#-#-###-#-####------####-##-#-##-###
 ####-#-#-###-#-####------####-##-#-##-###
#####-#-#-###-#-####----#-####-##-#-##-###
#####-#-#-###-#-#######-#-####-##-#-##-###
#####-#-#-###-#-#######-#-####-##-#-##-###
#####-#-#-#####-##############-##-#-##-###
#######-#-#####-##############-##-########
#######-#-#####-##############-##-########
#######-#-#####-##############-##-########
#######-#-####################-##-########
#########-#######################-########
##########################################
432
 #---------------#                
 #-------#--#----#                
 #-------#--#----#                
 #------##--#----#--------#------#
 #------##-##----#--------#-----##
 #------##-###--##--------#-----##
 #------##-###--##--------#-#---##
##------##-###--##--------#-#---##
##------##-###--##--------#-#---##
##------##-###--##--------#-#---##
##------##-###--##-------##-#--###
##-#---###-###--###---##-##-#--###
##-#---###-###--###---##-##-#--###
##-#---###-###--###---##-##-##-###
##-#---###-###--###---##-##-##-###
##-#---###-###--###---##-##-##-###
##-#---###-###--####-###-##-##-###
##-#---#######--####-###-##-##-###
##-#--########--####-###-##-##-###
##-#--########--########-#########
##-###########-#########-#########
##-###############################
##################################
323
     #-----#-----#--------#----##                   
#----#-----#-----#--------#----##-------#           
#----#-----#-----#----##--#----##-#-----#--#        
#----#--##-#-----#----##--#----##-#-----#-##----#   
#----##-##-#-----#-#--##--#----##-#----#####----#--#
#----##-##-#-----#-#--##--#----##-#--#-#####----#--#
#----##-##-#-----###--##--#----##-#-##-#####----#--#
#----##-##-#---#-###--##--#----##-#-##-#####---##--#
#----#####-#---#-###--##-##----##-#-##-######--##--#
#----#####-##--#-####-##-##----##-#-##-######-###--#
#----#####-##--######-##-##----##-#-##-##########-##
##---#####-##--######-##-##--#-##-#-##-##########-##
##---#####-##-#######-##-##--#-##-#-##-#############
##---#####-##-#######-#####--#-##-#-################
###--#####-##-#######-#####--####-##################
###--#####-##-#######-#####--####-##################
####-########-#######-######-####-##################
####-################-######-####-##################
####-###############################################
####################################################
360
